data/_ui/landing messages.txt

```diff
@@ -104,3 +104,11 @@
 	"planet/browndwarf-t-rogue"
 	"planet/browndwarf-y"
 	"planet/browndwarf-y-rogue"
+
+"landing message" "You cannot land on a nova!"
```

Comment:
```suggestion
"landing message" "You cannot land on a supernova remnant!"
```

Reply:
Let's just avoid this issue entirely.
```suggestion
"landing message" "This environment is too hostile for your ship to land in."
```

Reply:
It's a "stellar object". When the player moves slowly over a stellar object and presses the landing key, it is interpreted as an attempt to land on that object.

This object can be a planet, star, or anything else. Some of them are landable, some of them are not.

In this case, the entire gas cloud + central body is part of the same (star) stellar object. You can only try to land near the middle of it due to the radius of stars being limited to 80 pixels. So this is not a matter of having a central, landable body; it's just how the limited radius works out to something similar.